docs/fiddles/features/progress-bar/index.html

```diff
@@ -8,9 +8,8 @@
 <body>
     <h1>Hello World!</h1>
     <p>
-        We are using node <script>document.write(process.versions.node)</script>,
-        Chrome <script>document.write(process.versions.chrome)</script>,
-        and Electron <script>document.write(process.versions.electron)</script>.
+      Keep an eye on the progress bar for this application! It should advance to full and then
+      reset.
```

Comment:
```suggestion
      Keep an eye on the Dock (Mac)/Start Menu (Windows) icon for this application. It should indicate a progress that advances from 0 to 100% and then disappears.
```

Reply:
I like that suggestion, but I think you meant the "Task Bar (Windows)" instead of "Start Menu (Windows)".